q2_annotate/busco/busco.py

```diff
@@ -150,7 +283,33 @@ def _evaluate_busco(
         processing_func=_parse_busco_params, params=kwargs
     )
 
-    return _busco_helper(bins, common_args)
+    busco_results = _busco_helper(bins, common_args)
+    unbinned_percentage = {}
+    unbinned_count = {}   
+    # ### NEW Calculate unbinned percentage AFTER BUSCO RESULTS 
+    #add absolute count of unbinned contigs
+    if issubclass(type(bins), MultiMAGSequencesDirFmt):
+        ### NEW Add unbinned percentage per sample
+        if "unbinned_percentage" not in busco_results.columns:
```

Comment:
Why do you need this `if` (and the one under)? Is there a case where this column would already be present in the results?